pennylane_lightning/lightning_qubit.py

```diff
@@ -140,7 +141,7 @@ def apply_lightning(self, state, operations):
         apply(state_vector, op_names, op_wires, op_param, op_inverse, self.num_wires)
         return np.reshape(state_vector, state.shape)
 
-    def adjoint_jacobian(self, tape):
+    def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
```

Comment:
This is an update to the new `adjoint_jacobian` signature present in `QubitDevice` too.